bin/evm2wasm.js

```diff
@@ -76,4 +76,5 @@ try {
   })
 } catch (err) {
   console.error('Error: ' + err)
+  process.exit(-1)
```

Comment:
On POSIX usually 0 is used for success and 1 for failure. I think this may be returned as twos complement.

```
int main(int argc, char **argv) {
  return -1;
}
```

```
$ ./exit
$ echo $?
255
```

The same applies to nodejs.